README.md

```diff
@@ -30,14 +30,31 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.forEach --> doesn't return anything
```

Comment:
.forEach return each item in the array after it runs a function on it, please try to explain more clearly.